Documentation/GettingStartedWithTheMRTK.md

```diff
@@ -186,6 +186,13 @@ Here are some suggested next steps:
 * Learn about the [MRTK's Tools](../README.md#tools) that will empower your mixed reality design and development.
 * Read through [input simulation guide](InputSimulation/InputSimulationService.md) to learn how to simulate hand input in editor.
 
+## Getting help
+
+If you run into issues caused by MRTK or otherwise have questions about how to do something, there are a few resources that can help:
+
+* For questions, please reach out on either [StackOverflow](https://stackoverflow.com/questions/tagged/mrtk) or the [mixed-reality-toolkit channel](https://holodevelopers.slack.com/messages/C2H4HT858) on Slack. You can join the Slack community via the [automatic invitation sender](https://holodevelopersslack.azurewebsites.net/).
+* For bug reports, please [file an issue](https://github.com/microsoft/MixedRealityToolkit-Unity/issues/new/choose) on the GitHub repo.
```

Comment:
Do we actually want to put the "file an issue" first, because it's probably the area where we have the largest set of eyes? Or is my last statement not true in your opinion.

Reply:
Good call; I went ahead and flipped the order.
I originally ordered them in hopes that questions would be more common than bugs 😜 but I do think GitHub is the more visible spot to start

Reply:
Cool yeah - I have my own biases and knowledge which may differ from others, and in things like this I would prefer to defer to you.